src/test/java/com/es/phoneshop/dao/impl/ArrayListProductDaoTest.java

```diff
@@ -22,24 +26,43 @@
     private Product product2;
     @Mock
     private Product product3;
+    @Mock
+    private Product product4;
+
+    private final String query = "Apple n";
```

Comment:
Normally constants should be declared like this:
\<scope\> static final \<class\> CONSTANT_PARAMETER = \<value\>